backend/app/views/spree/layouts/admin.html.erb

```diff
@@ -4,30 +4,43 @@
     <%= render 'spree/admin/shared/head' %>
   </head>
 
-  <body class="admin">
+  <body class="admin <%= "new-layout" if @admin_layout %>">
```

Comment:
You can see `<%= @admin_layout.presence %>` be used below on the `content-wrapper`. This could have ended up on the `body` but the layout tiself is really just for the `content-wrapper`.